tracer/test/Datadog.Trace.Tests/Debugger/LiveDebuggerTests.cs

```diff
@@ -67,23 +58,24 @@ public void DebuggerDisabled_ServicesNotCalled()
 
         var settings = ImmutableDebuggerSettings.Create(tracerSettings);
         var discoveryService = new DiscoveryServiceMock();
-        var configurationPoller = new ConfigurationPollerMock();
+        var managerMock = new RemoteConfigurationManagerMock();
         var lineProbeResolver = new LineProbeResolverMock();
         var debuggerSink = new DebuggerSinkMock();
         var probeStatusPoller = new ProbeStatusPollerMock();
+        var updater = ConfigurationUpdater.Create(settings);
 
-        LiveDebugger.Create(settings, discoveryService, configurationPoller, lineProbeResolver, debuggerSink, probeStatusPoller);
+        var debugger = LiveDebugger.Create(settings, string.Empty, discoveryService, managerMock, lineProbeResolver, debuggerSink, probeStatusPoller, updater);
+        await Task.WhenAny(debugger.InitializeAsync(), Task.Delay(TimeSpan.FromSeconds(1)));
```

Comment:
I'm not sure I understand the logic here - we're waiting for the initialization to pass, or for one second to pass. Why do we need the 1 second delay and wouldn't this introduce flakiness?

Reply:
Fixed at feaec86d1c49441d56eb1bcee3b6f6be774e8b56